deployments/README.md

```diff
@@ -108,6 +108,11 @@ dashboard can be accessed at `http://localhost:9090`. The Grafana dashboard can
 Here is an example `.env` file which is used for production deployments and network updates. For
 security reasons it is not kept as part of version control. When deploying the network this
 file should be correctly populated and kept in the [`rialto`](./rialto) folder.
+The `UI_SUBSTRATE_PROVIDER` variable let you define the url of the nodes that the user interface
+will connect to. `UI_ETHEREUM_PROVIDER` is used only as a guidance for users to connect to
+Metamask to the right Ethereum network. `UI_EXPECTED_ETHEREUM_NETWORK_ID` should be set to the
+Ethereum network id. This is used by the user interface to prevent users from connecting their
+Metamask to an unexpected network.
```

Comment:
```suggestion
The `UI_SUBSTRATE_PROVIDER` variable lets you define the url of the Substrate node that the user interface
will connect to. `UI_ETHEREUM_PROVIDER` is used only as a guidance for users to connect
Metamask to the right Ethereum network. `UI_EXPECTED_ETHEREUM_NETWORK_ID`  is used by
the user interface as a fail safe to prevent users from connecting their Metamask extension to an
unexpected network.
```